quirks/websites-with-shared-credential-backends.json

```diff
@@ -351,6 +363,11 @@
         "seattle.bibliocommons.com",
         "spl.overdrive.com",
         "spl.org"
+    ],
+     [
+        "sfpl.bibliocommons.com",
+        "sfpl.overdrive.com",
+        "sfpl.org"
```

Comment:
Is there a login form hosted on `sfpl.org`? As far as I can tell it only redirects to `https://sfpl.bibliocommons.com/` for login, which wouldn't require an entry here.

Reply:
My mistake, you are correct.

Reply:
I'm new to GitHub, so forgive me for asking, but is there a way for me to fix this for your review?

Reply:
No worries! You did the right thing :) Just needed a small tweak to the JSON syntax which I made in https://github.com/apple/password-manager-resources/pull/402/commits/3ca3323727cc998d1a60da9cf9946d5239f0feb8